python-avd/pyavd/_eos_designs/ip_addressing/__init__.py

```diff
@@ -85,8 +85,13 @@ def mlag_ip_primary(self) -> str:
         """
         Return IP for MLAG Primary
 
+        If "mlag_peer_ipv4_address" is set, it is used.
         Default pool is "mlag_peer_ipv4_pool"
         """
+
+        if self._mlag_peer_ipv4_address:
+            return self._mlag_peer_ipv4_address
```

Comment:
same as below - this is not really correct to always pick this address for both primary and secondary